regression_tests/stages/build_platform_helper.sh

```diff
@@ -2,10 +2,18 @@
 
 set -e
 
-echo -e "\n\nBuild and install platform-helper\n"
+echo -e "\n\n### Build and install platform-helper\n"
```

Comment:
Trying to make the beginnings of each stage more obvious in the CodeBuild output.